draft-irtf-panrg-path-properties.md

```diff
@@ -308,6 +312,9 @@ Metrics such as loss patterns {{RFC3357}} and loss episodes {{RFC6534}} can be e
 If entities are basing policy or path selection decisions on path properties, they need to rely on the accuracy of path properties that other devices communicate to them.
 In order to be able to trust such path properties, entities may need to establish a trust relationship or be able to verify the authenticity, integrity, and correctness of path properties received from another entity.
 
+Entities that act based on their target path properties can negatively impact their privacy, e.g., if the target property leaks personal information about a user, such as his identity or which (type of) application is used.
```

Comment:
I don't like the usage of "communicate ... to the network" because that sounds to me more like an explicit (purposeful) action, i.e., the entity reveals the target property and the network chooses a suitable path.
With "act based on ...", I wanted to also cover the problem when choosing a certain path purely on the endpoint side, i.e., without involving the network in the path selection, which can have an impact on privacy because an observer in the network could infer from the chosen path(s) which application is used.

Maybe we can replace "communicate" with "reveal" which covers both cases?
```suggestion
Entities that reveal their target path properties to the network can negatively impact their own privacy, e.g., if the target property leaks personal information about a user, such as their identity or which (type of) application is used.
Such information could then allow network operators to block or re-prioritize traffic for certain users and/or application.
```

Reply:
Thanks, I like this phrasing!